www/md_base/src/icons/gear-outline.svg

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
There is also http://zavoloklom.github.io/material-design-iconic-font/icons.html

Problem of using the official bower package for material-design-icons is that it downloads the full set of png version, which is a painful 50MB. 

The full font is 50kb compressed, so I dont thing this is worth the pain of concatenating the svgs.